grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -1130,6 +1130,18 @@ def section_streaming(outer_panels):
                         ),
                     ],
                 ),
+                panels.timeseries_epoch(
+                    "Current Epoch of Materialize Views",
+                    "The current epoch that the Materialize Executors are processing. If an MV's epoch is far behind the others, "
+                    "it's very likely to be the performance bottleneck",
```

Comment:
BTW, if an upstream MV is blocked, then all downstream MVs will have epoch lag? (Therefore we can't locate the root cause from this metrics alone) 🤔

Reply:
I found you once raised the idea to show the epoch on DAG. 🤔

https://github.com/risingwavelabs/risingwave/issues/13481#issuecomment-2301341892